domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -103,11 +103,15 @@
     System.setProperty(LoggerFactory.LOGGER_DELEGATE_FACTORY_CLASS_NAME, "io.vertx.core.logging.Log4j2LogDelegateFactory");
   }
 
+  private int QUERY_TIMEOUT = 0;
+
+/*
   @BeforeClass
   public static void doesNotCompleteOnWindows() {
     final String os = System.getProperty("os.name").toLowerCase();
     org.junit.Assume.assumeFalse(os.contains("win")); // RMB-261
   }
+*/
```

Comment:
Can you either remove `/*` and `*/`? You may create a separate issue and PR to completely remove `doesNotCompleteOnWindows()`.

Reply:
done